helper/release_enarx.sh

```diff
@@ -343,10 +339,20 @@ on_exit() {
 }
 trap on_exit EXIT
 
+
+if [[ "$FAST" == "true" ]]; then
+    echo -e "Warning: FAST environment variable set, will only test with single image."
+    IMAGES=( "${IMAGES_FAST[@]}" )
+else
+    shout "THOROUGH environment variable set, will test with all images. This will take up to 90 minutes for the entire process"
```

Comment:
Better to say "FAST environment variable not set", since this might encourage someone to try setting a THOROUGH environment variable which doesn't get checked at all.

Reply:
Fixed